src/caret_analyze/plot/visualize_lib/bokeh/timeseries.py

```diff
@@ -66,20 +67,51 @@ def create_figure(self) -> Figure:
         # Apply xaxis offset
         records_range = Range([to.to_records() for to in target_objects])
         frame_min, frame_max = records_range.get_range()
-        if self._xaxis_type == 'system_time':
+        converter: Optional[ClockConverter] = None
+        if self._xaxis_type == 'sim_time':
+            # TODO: refactor
+            # get converter
+            if isinstance(target_objects[0], Communication):
+                for comm in target_objects:
+                    assert isinstance(comm, Communication)
+                    if comm.callback_subscription:
+                        converter_cb = comm.callback_subscription
+                        converter = converter_cb._provider.get_sim_time_converter()
+                        break
+            else:
+                converter = target_objects[0]._provider.get_sim_time_converter()
+        if converter:
+            frame_min_convert = converter.convert(frame_min)
+            frame_max_convert = converter.convert(frame_max)
+            x_range_name = 'x_plot_axis'
+            apply_x_axis_offset(fig, frame_min_convert, frame_max_convert, x_range_name)
```

Comment:
```suggestion
            frame_min = converter.convert(frame_min)
            frame_max = converter.convert(frame_max)
            x_range_name = 'x_plot_axis'
            apply_x_axis_offset(fig, frame_min, frame_max, x_range_name)
```

Reply:
The variables "frame_min" and "frame_max" are of type "int," and the return value of "converter.convert()" is of type "float." An error occurred in "test.mypy." How about the following modifications?
```
            frame_min = int(converter.convert(frame_min))
            frame_max = int(converter.convert(frame_max))
            x_range_name = 'x_plot_axis'
            apply_x_axis_offset(fig, frame_min, frame_max, x_range_name)
```

Reply:
From the difference between the return values of `records_range.get_range()` and `converter.convert()`, it is better to have another variable for `flame_min/max` and `flame_min/max_contert`.

Hence, there is no need to apply the above suggestion.